packages/api-explorer/src/ResponseSchema.jsx

```diff
@@ -87,6 +87,14 @@ class ResponseSchema extends React.Component {
     const { operation, oas } = this.props;
     if (!operation.responses || Object.keys(operation.responses).length === 0) return null;
     const schema = this.getSchema(operation);
+
+    let response = operation.responses[this.state.selectedStatus];
+
+    // @todo This should really be called higher up when the OAS is processed within the Doc component.
+    if (response.$ref) {
```

Comment:
I don't particularly like doing this schema definition lookup here and in `create-code-shower`, but moving it further up the stack is going to require a heavy refactor.